infra/base-images/base-builder/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apt-get install -y git subversion jq python3 zip make
 
 # Default build flags for various sanitizers.
 ENV SANITIZER_FLAGS_address "-fsanitize=address -fsanitize-address-use-after-scope"
-ENV SANITIZER_FLAGS_undefined "-fsanitize=bool,function,signed-integer-overflow,shift,vla-bound,vptr -fno-sanitize-recover=undefined"
+ENV SANITIZER_FLAGS_undefined "-fno-sanitize-recover=alignment,bool,bounds,enum,float-divide-by-zero,function,integer-divide-by-zero,null,object-size,shift,signed-integer-overflow,vla-bound,vptr"
```

Comment:
Actually I'd rather have `-fsanitize=`, since we always set `UBSAN_OPTIONS=halt_on_error=1`, but it feels that `-fno-sanitize-recover=` might be easier for local reproduction, as it always terminates the program (no need to have `UBSAN_OPTIONS`).